src/routes/assets.ts

```diff
@@ -70,11 +70,30 @@ function register(app: express.Application) {
     app.use(`/${assetPath}/node_modules/split.js/dist/`, persistentCacheStatic(path.join(srcRoot, '..', 'node_modules/split.js/dist/')));
 
     app.use(`/${assetPath}/node_modules/panzoom/dist/`, persistentCacheStatic(path.join(srcRoot, '..', 'node_modules/panzoom/dist/')));
-
     // i18n
     app.use(`/${assetPath}/node_modules/i18next/`, persistentCacheStatic(path.join(srcRoot, "..", 'node_modules/i18next/')));
     app.use(`/${assetPath}/node_modules/i18next-http-backend/`, persistentCacheStatic(path.join(srcRoot, "..", 'node_modules/i18next-http-backend/')));
     app.use(`/${assetPath}/translations/`, persistentCacheStatic(path.join(srcRoot, "public", "translations/")));
+
+    // Seems work without this, not sure will cause other bugs. Maybe can test without this for a while.
+    // app.use(`/${assetPath}/node_modules/eslint/bin/`, persistentCacheStatic(path.join(srcRoot, '..', 'node_modules/eslint/bin/')));
```

Comment:
ESLint is required for linting frontend and backend script notes, without it JavaScript errors are not displayed and there is an error in the console:

![image](https://github.com/user-attachments/assets/667dcd53-4b39-4b28-9ed7-f7b7d349943b)

Reply:
It causes some errors after I add eslint path into assets, but it fixed if I comment out eslint in `libraries/codemirror/addon/lint/eslint.js`. Will remove this also in this PR.
![image](https://github.com/user-attachments/assets/4223fef3-51a2-4b27-a210-eb14ab7c7797)